opencog/spacetime/octomap/OctoMapValue.cc

```diff
@@ -14,6 +14,11 @@ void OctoValue::update() const
     _value = {it->x(), it->y(), it->z()};
 }
 
+void OctoValue::update(const std::vector<double>& loc) const
+{
```

Comment:
Values are meant to be externally immutable. So again, no calls of this form. Again, one of the primary drivers is thread safety.

Reply:
Noted.